torchserve-vicuna/grpc_client.py

```diff
@@ -0,0 +1,149 @@
+import argparse
+
+import grpc
+import inference_pb2
+import inference_pb2_grpc
+import management_pb2
+import management_pb2_grpc
+
+
+def get_inference_stub():
```

Comment:
Why do we need to copy the grpc_client.py here? Can we not use the TorchServe version from: https://github.com/pytorch/serve/blob/master/ts_scripts/torchserve_grpc_client.py